docs/rbs_by_example.md

```diff
@@ -6,20 +6,20 @@ The purpose of this doc is to teach you how to write RBS signatures by using the
 
 ## Examples
 
+In each example, the first snippet is for *Ruby* and the second one is for *RBS*.
```

Comment:
[note] I thought `# .rb` and `# .rbs` in snippets are redundant, so I've removed them and added the description above.